### ISSUE TITLE
Cannot set an arbitrary DID as assignee to an issue

### ISSUE THREAD

**Issue by gsaslis** [cytechmobile/radicle-jetbrains-plugin#346]
The assignee "edit" button, shows a dropdown list that allows me to select some DIDs, **but** does not offer the option to set a different DID that isn't shown on the list. 

It is fine to keep the dropdown, for easy access, but it should also be possible to assign another DID. 

As a final question, how is the list populated?

**Comment by Stelios123** [comment]
So you want something like the Label Select right? We are getting the list from this URL `/api/v1/projects/z9z8YSabLQukgeXNLeTC54ZGLjiF`

**Comment by Stelios123** [comment]
Do we still want this? As I said you can't assign a delegate who doesn't exist in delegates ( API response)

**Comment by gsaslis** [comment]
hmmm so you're saying the API endpoint only accepts delegates as assignees? 

That doesn't sound quite right 🤔 

e.g. in Heartwood, there is only one delegate but it _should!_ be possible to assign other users, as multiple folks are working on the project.

**Comment by Stelios123** [comment]
ok let me try again and i will come back
